types/ember-testing-helpers/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "allowSyntheticDefaultImports": true
```

Comment:
It *does* have an effect in this case, because I am importing it in the test this way (since that's the primary way most consumers will use it) – though I can switch it over to the other if that's preferred.